clang/lib/CodeGen/CGExprConstant.cpp

```diff
@@ -2061,6 +2071,58 @@ ConstantLValueEmitter::VisitCallExpr(const CallExpr *E) {
   }
 }
 
+ConstantLValue
+ConstantLValueEmitter::emitPointerAuthSignConstant(const CallExpr *E) {
```

Comment:
Names to fix: `unsignedPointer`, `key`, `storageAddress`, `otherDiscriminator`, `signedPointer`